### PR TITLE
Feature#49 fix margin

### LINKED ISSUE #49: Fix margi in views chose role
# REQUIREMENTS
Reduce the margin above in the container that is above the footer

# Acceptance Criteria
- [ ] Reduce margin

### PR DESCRIPTION
**Pull request by RobertoCarMed**
# Description
What did you implemented/Why did you implemented this:
 - Fixed margins of main container in select record view.
 - Fixed the footer so that it is always shown at the bottom of the page

resolve #49

### CODE REVIEW

**Review by oramasangel — APPROVED** [review]

File: app/assets/stylesheets/teachers/components/_index_desktop.scss [hunk 1]
```diff
@@ -1,3 +1,9 @@
 .card-teacher{
   flex-wrap: wrap;
 }
+
+// html > body > .body{
+//   justify-content: start;
+//   min-height: 87vh;
+
+// }
```
Comment (SampabloC) [inline app/assets/stylesheets/teachers/components/_index_desktop.scss:9]: If you don't need it you can delete it.

**Review by SampabloC — APPROVED** [review]

**Review by xFerchoVJ — APPROVED** [review]
LGTM

File: app/assets/stylesheets/teachers/components/_index_desktop.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (RobertoCarMed) [reply app/assets/stylesheets/teachers/components/_index_desktop.scss:9]: ok, thank you

Comment (RobertoCarMed) [reply app/assets/stylesheets/teachers/components/_index_desktop.scss:9]: ok, thank you

**Pull request merged**